Include disk usage (Data.fs and blobstorage) in stats

lukasgraf opened the pull request:
Disk usage stats are gathered using the `du` utility. In order to avoid running an I/O heavy and potentially long running process like `du` from inside a Zope instance, this is being done from the `bin/dump-content-stats` wrapper script.

This script invokes `du`, stores the results in a JSON file, and then triggers the `@@dump-content-stats` view as usual.

That view then picks up the disk usage stats from that file (if present), and includes them in the dumped stats.

Closes 4teamwork/opengever.core#4675

buchi commented on ftw/contentstats/demo/filters.py:
```diff
@@ -33,3 +34,17 @@ def keep(self, key):
+        self.request = request
+
+    def keep(self, key):
+        if basename(key).endswith('.tmp'):
```
With the current implementation can this ever be true?
The paths are limited to `var/blobstorage` and `var/filestorage/Data.fs` or do I misunderstand something?

lukasgraf commented:
Updated, (mostly) as discussed:
- Changed size unit to bytes instead of kilobytes
- Calculate separate `total` (entire deployment directory, count hard links only once) and `subtree` (sizes of individual subdirs / files in the deployment directory, apparent size, count hard links multiple times) sizes 
- Collect disk usage data in a form that could later also be used by `agent.smith`
- Don't attempt to use *BSD/GNU portable `du` options any more, instead use the GNU `gdu` from coreutils on Mac OS if available
